apps/src/music/views/MusicLabView.tsx

```diff
@@ -115,8 +120,16 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
         };
         return {level_data: updatedLevelData};
       });
+    } else if (isToolboxMode) {
+      header.showLevelBuilderSaveButton(() => {
```

Comment:
When the toolbox mode is opened, 'Not saved' is displayed in the header. If a content editor saves the toolbox, 'Saved' is displayed. However, when the toolbox workspace is updated, there is no indication that the current state of workspace is saved. Should we display 'Not saved' at this point?

Reply:
Maybe? This component was borrowed from other labs. I think this behavior is consistent if not ideal.